release_notes/Service_Fabric_ReleaseNotes_82CU3.md

```diff
@@ -48,7 +48,7 @@ Microsoft Azure Service Fabric 8.2 Cumulative Update 3.0 Release Notes
 
 | Versions | IssueType | Description | Resolution | 
 |-|-|-|-|
-| **Windows - 8.2.1620.9590<br>Ubuntu 16 - 8.2.1434.1<br>Ubuntu 18 - 8.2.1434.1** | **Bug** | Backup Restore Service failure| **Brief desc**: When large number of backup policy addition requests are made in parallel and RetentionManager object is created in a different thread,RunAsync is cancelled for reconfiguration and gets stuck at DisposeRetentionManager waiting for RetentionManagerSemaphore to be released.<br>**Impact**: Backup Restore Service will fail blocking cluster upgrade.<br>**Fix**: Release the RetentionManagerSemaphore properly. 
+| **Windows - 8.2.1620.9590<br>Ubuntu 16 - 8.2.1434.1<br>Ubuntu 18 - 8.2.1434.1** | **Bug** | Backup Restore Service failure| **Brief desc**: when large number of backup policy addition requests are made in parallel and RetaintionManager object created in a different thread,RunAsync gets cancelled for reconfiguration due to DisposeRetentionManager waiting for RetentionManagerSemaphore to be released.<br>**Impact**: Backup Restore Service will fail blocking cluster upgrade.<br>**Fix**: Release the RetentionManagerSemaphore properly. 

```

Comment:
minor edit suggestion
When a large number of backup policy addition requests are made in parallel and RetentionManager object is created in a different thread, RunAsync gets cancelled for reconfiguration due to DisposeRetentionManager waiting for RetentionManagerSemaphore to be released.